src/api/res.py

```diff
@@ -33,6 +33,7 @@ def res(
     stat: Union[float, int, str, Decimal, None] = None,
     sigfigs: Union[int, None] = None,
     decimal_places: Union[int, None] = None,
+    print: Union[bool, None] = None,  # pylint: disable=redefined-builtin
```

Comment:
Instead of just disabling this error message, shouldn't we rename `print` to something different to avoid overriding the internal `print` method accidentally? E.g. use `should_print`.